src/main/java/com/wetwater/washwater/item/BucketMechanics.java

```diff
@@ -39,6 +42,51 @@ public static boolean creativePipettePlace(Level level, BlockPos pos, ItemStack
             }
         return true;
```

Comment:
Move all code in this class to thier relevant item classes, it doesn't make much sense to put it here instead. I don't think its ever called from somewhere else right?